src/nft/components/icons.tsx

```diff
@@ -667,7 +670,7 @@ export const ActivityTransferIcon = (props: SVGProps) => (
   </svg>
 )
 
-export const ActivityExternalLinkIcon = (props: SVGProps) => (
+export const ActivityExternalLinkIcon = (_props: SVGProps) => (
```

Comment:
nit: can we just remove it?

Reply:
i just wanted to keep the pattern so someone knows they _could_ use the props in the future